Automattic/wp-calypso
# Add domain notices to the redesigned domains list pages

#### Changes proposed in this Pull Request

This PR creates the redesigned notices with the new messages and statuses - as described in p1638882214174600-slack-C020919TDRA) for the domain list. 

This is part of the domain settings pages redesign project detailed in pcYYhz-m2-p2.

#### Preview - all possible status
![image](https://user-images.githubusercontent.com/18705930/148577243-710987ff-3ebb-4fd5-9388-0a3821251cb7.png)

#### Testing instructions
- Build this branch locally or open the live Calypso link
  - If you're in the live Calypso link, please append `?flags=domains/settings-page-redesign` to your URL to enable the appropriate feature flag
- Go to "Upgrades > Domains"
- Make sure that the notices are correctly displayed following the same logic described in p1638882214174600-slack-C020919TDRA.
- Confirm that all the original features of the "old" management pages are still working as intended.

**P.S: Please note that it was not possible to implement some of the statuses in the notices. We will eventually move the controlling logic for them somewhere else so these remaining statuses can be handled.**
